### PR TITLE
Add Lumen instruction

### PR DESCRIPTION
**Pull request by adrian7**
I had to do some googling before installing this on Lumen - so here are the steps I took.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,10 @@ Install
     ```bash
     composer require baopham/dynamodb
     ```
+* For Lumen only - additional steps
+  - enable eloquent support in bootstrap/app.php: uncomment/add `$app->withEloquent()` just after `$app = new Laravel\Lumen\Application ...`
```
Comment (baopham) [inline README.md]: * Title case "Eloquent"
* Quote bootstrap/app.php with backticks: `bootstrap/app.php`

File: README.md [hunk 2]
```diff
@@ -44,6 +44,10 @@ Install
...
+  - set up a `config_path` helper: https://gist.github.com/mabasic/21d13eab12462e596120
```
Comment (baopham) [inline README.md]: * Make this a link

```md
- set up a [`config_path` helper](https://gist.github.com/mabasic/21d13eab12462e596120)
```

File: README.md [hunk 3]
```diff
@@ -44,6 +44,10 @@ Install
...
+  - install `vendor:publish` command: https://github.com/laravelista/lumen-vendor-publish
```
Comment (baopham) [inline README.md]: Same here, make this a link

```md
- install [`vendor:publish` command](https://github.com/laravelista/lumen-vendor-publish)
```

File: README.md (hunk 2 above)
Comment (baopham) [inline README.md]: Looks like you just need to do `$app->configure('dynamodb');`? See #134

Comment (adrian7) [reply README.md]: Yes, but that file need to be copied from the src folder of the package.  Manually or using `vendor-publish`. I prefer the latter, hence more steps.

File: README.md [hunk 4]
```diff
@@ -55,13 +55,37 @@ Install
...
+    
+* Laravel
 
     ```php
+    php artisan vendor:publish
+    ```    
+    
+* Lumen
+  
+  - Set up a `config_path` helper: https://gist.github.com/mabasic/21d13eab12462e596120
```
Comment (gchauqui-bondacom) [inline README.md]: Or you can fix `DynamoDbServiceProvider` replacing `config_path()` to `app()->path()`. It works in Lumen and Laravel :+1:

Comment (amazzoccone) [reply README.md]: Thanks @gchauqui-bondacom ! `app()->path()` makes it works. It should be cool if the pakage change this so then itsn't necessary to register `config_path` helper. What do you think @baopham  ?

Comment (baopham) [reply README.md]: yep, PR is welcome.

**Comment by baopham** [comment]
Thank you

**Pull request merged**